nisystemlink/clients/core/_enterprise_http_configuration.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of EnterpriseHttpConfiguration."""
+
+from nisystemlink.clients import core
+
+
+class EnterpriseHttpConfiguration(core.HttpConfiguration):
```

Comment:
We use the term "enterprise" as a marker for the tests as well, what should we do about that case?

Reply:
I think test markers and parameters are fine since they don't appear in the API, just the test code. We can always rename them later if a different name makes more sense.